### ISSUE TITLE
Implement CMS

### ISSUE THREAD

**Issue by Germano123** [gameguild-gg/website#52]
Create CMS Editor and renderer to manage all contents.

- [ ] Courses management system: 
- [ ] Blog posts: #2 
- [ ] Projects versions, pages; and versions: #84 
- [ ] Manage translations: #92 
- [ ] Content Editor - Lexical
- [ ] Smart Editor using AI to give sugestions

**Comment by tolstenko** [comment]
@Germano123 are you going to do this?

**Comment by Germano123** [comment]
Yes, I was late with the delivery but I just need to make a cherry pick based on what was discussed in the last alignment.

**Comment by tolstenko** [comment]
i am doing this now. I already did the front-end.

**Comment by tolstenko** [comment]
It is already deployed the front with mockups at https://gameguild.gg/courses

**Comment by tolstenko** [comment]
I edited your original post to be more specific.

**Comment by tolstenko** [comment]
@migmoroni create a new issue to reflect this better, given you are working on this now.
